plugins/hls-class-plugin/test/Main.hs

```diff
@@ -96,7 +96,7 @@ codeLensTests recorder = testGroup
     , goldenCodeLens recorder "Apply code lens for local class" "LocalClassDefine" 0
     , goldenCodeLens recorder "Apply code lens on the same line" "Inline" 0
     , goldenCodeLens recorder "Don't insert pragma while existing" "CodeLensWithPragma" 0
-    , onlyRunForGhcVersions [GHC92] "Only ghc-9.2 enabled GHC2021 implicitly" $
+    , onlyRunForGhcVersions [GHC92, GHC94] "Only ghc-9.2 enabled GHC2021 implicitly" $
```

Comment:
As CI failed, let me add one nitpicking here. The message may sound better with `"ghc-9.2 or above"` instead of `"ghc-9.2"`.